Assets/Scenes/Levels/Level_1/NavMesh-Navigation Plane 1.asset.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
Do we need a `NavMesh-Navigation Plane.asset` and a `NavMesh-Navigation Plane.asset` for each level? Or was one intended to be for a backup or experiment?

Reply:
Yes, one is for Sheep and one is for Dog. This is useful for filtering NavMesh agent types for jumping fences. I am looking into how to combine the NavMeshes or reduce their size.